src/docs/release-notes/R26/Summary.mdx

```diff
@@ -1 +1 @@
-Migrated our code base to a [new location](https://github.com/etn-ccis?q=blui); added guidelines for [app store logos](/style/logo), [buttons](/patterns/buttons), and [UI design audit](/design/design-audit); added support for Angular 14.
+Migrated our code base to a [new location](https://github.com/etn-ccis?q=blui), added guidelines for [app store logos](/style/logo), [buttons](/patterns/buttons), and [UI design audit](/design/design-audit), added support for Angular 14.
```

Comment:
```suggestion
Migrated our code base to a [new location](https://github.com/etn-ccis?q=blui), added support for Angular 14, and added guidelines for [app store logos](/style/logo), [buttons](/patterns/buttons), and [UI design audit](/design/design-audit).
```

Reply:
@daileytj that looks more organized, I've added it in the new commit